packages/kolibri-common/strings/enhancedQuizManagementStrings.js

```diff
@@ -18,7 +18,7 @@ export const enhancedQuizManagementStrings = createTranslator('EnhancedQuizManag
     message: 'Quiz title',
   },
   addQuizSections: {
-    message: 'Add one or more sections to the quiz, according to your needs',
+    message: 'Add one or more sections, according to your needs',
```

Comment:
My general thought about this string, is whether it is needed at all.

Reply:
I have the same thought

Reply:
I have no objections to eliminate this string! 💥